### PR TITLE
Sdk major release 2020

### PR DESCRIPTION
**Pull request by kevinkowa**
BREAKING CHANGE: api def tag 'sdk-major-release-2020' gen tag 3.19.0

### CODE REVIEW

**Review by germanattanasio — APPROVED** [review]
LGTM

File: speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java [hunk 1]
```diff
@@ -215,7 +215,7 @@ public void testGetModelNoOptions() throws Throwable {
...
...\"confidence\": 0, \"timestamps\": [\"timestamps\"], \"word_confidence\": [\"wordConfidence\"]}], \"keywords_result\": {\"mapKey\": [{\"normalized_text\": \"normalizedText\", \"start_time\": 9, \"end_time\": 7, \"confidence\": 0}]}, \"word_alternatives\": [{\"start_time\": 9, \"end_time\": 7, \"alternatives\": [{\"confidence\": 0, \"word\": \"word\"}]}], \"end_of_utterance\": \"end_of_data\"}], \"result_index\": 11, \"speaker_labels\": [{\"from\": 4, \"to\": 2, \"speaker\": 7, \"confidence\": 10, \"final\": true}], \"processing_metrics\": {\"processed_audio\": {\"received\": 8, \"seen_by_engine\": 12, \"transcription\": 13, \"speaker_labels\": 13}, \"wall_clock_since_first_byte_received\": 31, \"periodic\": true}, \"audio_metrics\": {\"sampling_interval\": 16, \"accumulated\": {\"final\": true, \"end_time\": 7, \"signal_to_noise_ratio\": 18, \"speech_ratio\": 11, \"high_frequency_loss\": 17, \"direct_current_offset\": [{\"begin\": 5, \"end\": 3, \"count\": 5}], \"clipping_rate\": [{\"begin\": 5, \"end\": 3, \"count\": 5}], \"speech_level\": [{\"begin\": 5, \"end\": 3, \"count\": 5}], \"non_speech_level\": [{\"begin\": 5, \"end\": 3, \"count\": 5}]}}, \"warnings\": [\"warnings\"]}";
```
Comment (mediumTaj) [inline speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java:218]: Can you check this? the timestamps and word confidence should be an array of arrays.

File: speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java [hunk 2]
```diff
@@ -425,7 +425,7 @@ public void testUnregisterCallbackNoOptions() throws Throwable {
...
...": [\"timestamps\"], \"word_confidence\": [\"wordConfidence\"]}], \"keywords_result\": {\"mapKey\": [{\"normalized_text\": \"normalizedText\", \"start_time\": 9, \"end_time\": 7, \"confidence\": 0}]}, \"word_alternatives\": [{\"start_time\": 9, \"end_time\": 7, \"alternatives\": [{\"confidence\": 0, \"word\": \"word\"}]}], \"end_of_utterance\": \"end_of_data\"}], \"result_index\": 11, \"speaker_labels\": [{\"from\": 4, \"to\": 2, \"speaker\": 7, \"confidence\": 10, \"final\": true}], \"processing_metrics\": {\"processed_audio\": {\"received\": 8, \"seen_by_engine\": 12, \"transcription\": 13, \"speaker_labels\": 13}, \"wall_clock_since_first_byte_received\": 31, \"periodic\": true}, \"audio_metrics\": {\"sampling_interval\": 16, \"accumulated\": {\"final\": true, \"end_time\": 7, \"signal_to_noise_ratio\": 18, \"speech_ratio\": 11, \"high_frequency_loss\": 17, \"direct_current_offset\": [{\"begin\": 5, \"end\": 3, \"count\": 5}], \"clipping_rate\": [{\"begin\": 5, \"end\": 3, \"count\": 5}], \"speech_level\": [{\"begin\": 5, \"end\": 3, \"count\": 5}], \"non_speech_level\": [{\"begin\": 5, \"end\": 3, \"count\": 5}]}}, \"warnings\": [\"warnings\"]}], \"warnings\": [\"warnings\"]}";
```
Comment (mediumTaj) [inline speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java:428]: Can you check this? the timestamps and word confidence should be an array of arrays.

File: speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java [hunk 3]
```diff
@@ -542,7 +542,7 @@ public void testCreateJobNoOptions() throws Throwable {
...
... [\"timestamps\"], \"word_confidence\": [\"wordConfidence\"]}], \"keywords_result\": {\"mapKey\": [{\"normalized_text\": \"normalizedText\", \"start_time\": 9, \"end_time\": 7, \"confidence\": 0}]}, \"word_alternatives\": [{\"start_time\": 9, \"end_time\": 7, \"alternatives\": [{\"confidence\": 0, \"word\": \"word\"}]}], \"end_of_utterance\": \"end_of_data\"}], \"result_index\": 11, \"speaker_labels\": [{\"from\": 4, \"to\": 2, \"speaker\": 7, \"confidence\": 10, \"final\": true}], \"processing_metrics\": {\"processed_audio\": {\"received\": 8, \"seen_by_engine\": 12, \"transcription\": 13, \"speaker_labels\": 13}, \"wall_clock_since_first_byte_received\": 31, \"periodic\": true}, \"audio_metrics\": {\"sampling_interval\": 16, \"accumulated\": {\"final\": true, \"end_time\": 7, \"signal_to_noise_ratio\": 18, \"speech_ratio\": 11, \"high_frequency_loss\": 17, \"direct_current_offset\": [{\"begin\": 5, \"end\": 3, \"count\": 5}], \"clipping_rate\": [{\"begin\": 5, \"end\": 3, \"count\": 5}], \"speech_level\": [{\"begin\": 5, \"end\": 3, \"count\": 5}], \"non_speech_level\": [{\"begin\": 5, \"end\": 3, \"count\": 5}]}}, \"warnings\": [\"warnings\"]}], \"warnings\": [\"warnings\"]}]}";
```
Comment (mediumTaj) [inline speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java:545]: Can you check this? the timestamps and word confidence should be an array of arrays.

File: speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java [hunk 4]
```diff
@@ -581,7 +581,7 @@ public void testCheckJobsWOptions() throws Throwable {
...
...": [\"timestamps\"], \"word_confidence\": [\"wordConfidence\"]}], \"keywords_result\": {\"mapKey\": [{\"normalized_text\": \"normalizedText\", \"start_time\": 9, \"end_time\": 7, \"confidence\": 0}]}, \"word_alternatives\": [{\"start_time\": 9, \"end_time\": 7, \"alternatives\": [{\"confidence\": 0, \"word\": \"word\"}]}], \"end_of_utterance\": \"end_of_data\"}], \"result_index\": 11, \"speaker_labels\": [{\"from\": 4, \"to\": 2, \"speaker\": 7, \"confidence\": 10, \"final\": true}], \"processing_metrics\": {\"processed_audio\": {\"received\": 8, \"seen_by_engine\": 12, \"transcription\": 13, \"speaker_labels\": 13}, \"wall_clock_since_first_byte_received\": 31, \"periodic\": true}, \"audio_metrics\": {\"sampling_interval\": 16, \"accumulated\": {\"final\": true, \"end_time\": 7, \"signal_to_noise_ratio\": 18, \"speech_ratio\": 11, \"high_frequency_loss\": 17, \"direct_current_offset\": [{\"begin\": 5, \"end\": 3, \"count\": 5}], \"clipping_rate\": [{\"begin\": 5, \"end\": 3, \"count\": 5}], \"speech_level\": [{\"begin\": 5, \"end\": 3, \"count\": 5}], \"non_speech_level\": [{\"begin\": 5, \"end\": 3, \"count\": 5}]}}, \"warnings\": [\"warnings\"]}], \"warnings\": [\"warnings\"]}";
```
Comment (mediumTaj) [inline speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java:584]: Can you check this? the timestamps and word confidence should be an array of arrays.

**Review by mediumTaj — CHANGES_REQUESTED** [review]
Looks good overall, can you check to see if the handwritten change has  been applied to timestamps and word confidence? They should be an array of arrays.

File: speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java (hunk 1 above)
Comment (kevinkowa) [reply speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java]: fixed in [cde7e20](https://github.com/watson-developer-cloud/java-sdk/pull/1143/commits/cde7e20e3789bedd96591f6ae4026b8b9b37e8eb)

File: speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java (hunk 3 above)
Comment (kevinkowa) [reply speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java]: fixed in [cde7e20](https://github.com/watson-developer-cloud/java-sdk/pull/1143/commits/cde7e20e3789bedd96591f6ae4026b8b9b37e8eb)

File: speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java (hunk 2 above)
Comment (kevinkowa) [reply speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java]: fixed in [cde7e20](https://github.com/watson-developer-cloud/java-sdk/pull/1143/commits/cde7e20e3789bedd96591f6ae4026b8b9b37e8eb)

File: speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java (hunk 4 above)
Comment (kevinkowa) [reply speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextTest.java]: fixed in [cde7e20](https://github.com/watson-developer-cloud/java-sdk/pull/1143/commits/cde7e20e3789bedd96591f6ae4026b8b9b37e8eb)

**Review by mediumTaj — APPROVED** [review]
👍 looks good! Once this is merged please create a PR to master for final review

**Pull request merged**